less/navigation.less

```diff
@@ -1 +1,58 @@
-// Navigation Styles here
\ No newline at end of file
+// Navigation Styles here
+
+.navigation {
+    display: flex;
+    width: 100%;
+    border-bottom: 2px dashed @navigation-border;
+    justify-content: space-evenly;
+    padding: 2%;
+
+    nav {
+        display: flex;
+        font-size: 1.5rem;
+        width: 800px;
+        padding-right: 20%;
```

Comment:
Padding right would not be needed here if container class is added to parent div. Width should be set to 100%.